api/api.yaml

```diff
@@ -119,13 +136,43 @@ components:
           maximum: 9223372036854775807
 
     LineageCreationResponse:
-      type: object
+      required:
+        - id
+        - extId
       properties:
         id:
           type: string
         extId:
           type: string
 
+    LineageGetResponse:
+      required:
+        - id
+        - extId
+        - nextNonce
+        - leasedNonceCount
+        - releasedNonceCount
+        - maxLeasedNonceCount
+        - maxNonceValue
+        - version
+      properties:
+        id:
+          type: string
+        extId:
+          type: string
+        nextNonce:
+          type: integer
+        leasedNonceCount:
+          type: integer
+        releasedNonceCount:
+          type: integer
+        maxLeasedNonceCount:
+          type: integer
+        maxNonceValue:
+          type: integer
+        version:
```

Comment:
i think this should be internal

Reply:
Fixed by 20b118cd903b5255e354ce69258470bf84de7bd4